evil-vars.el

```diff
@@ -1280,6 +1280,11 @@ having higher priority.")
 (defvar evil-command-properties nil
   "Specifications made by `evil-define-command'.")
 
+(defvar evil-change-commands '(evil-change)
+  "Commands that wrap or replace `evil-change'.
+This list exists to apply an inconsistency with vim's change command
+to commands that wrap or redefine it. See emacs-evil/evil#916.")
```

Comment:
@jgkamat @wasamasa 

"See emacs-evil/evil#916." 
should be
"See emacs-evil/evil/issues/916." or "See github.com/emacs-evil/evil/issues/916."

Reply:
Markdown formatting or shortcuts messing up the formatting intended. Putting it in code blocks.

``
"See emacs-evil/evil#916." 
should be
"See emacs-evil/evil/issues/916." or "See github.com/emacs-evil/evil/issues/916."
``

Reply:
I don't think `emacs-evil/evil/issues/916` would be an improvement, but I do think a full link could be, ie: `https://github.com/emacs-evil/evil/issues/916`. I'm not sure if there's a convention for linking to issues in source files, but we should probably follow that convention if it exists (in evil, or in general).

Reply:
Sure. Wasn't sure. When I went to github.com/emacs-evil/evil#916 it went directly to the root project. I wasn't sure if it referred to an Issue or a Pull Request. Thought that the link clarification would make it clearer. Not sure of "a convention" either, but clarity with regards to the point of reference is nice.

Reply:
Judging from existing code, the convention is `#916`.  I'd only add the project name if it cross-references a different one (like, `goto-chg.el` for example).